lib/taskbuilder.js

```diff
@@ -375,7 +389,30 @@ exports.build = function (ops) {
         groupItems;
 
     if (ops.command === "group") {
-        return buildGroup(ops);
+        var groupItems;
+
+        // get the group data from the json file
+        try {
+            groupItems = require(GROUPS_DIR + "/" + ops.commandValue + ".json");
+        } catch (e) {
+            return Promise.reject("Couldn't find group with name " + ops.commandValue);
+        }
+
+        return buildGroup(groupItems, ops);
+
+    } else if (ops.command === "build-group") {
+        return childProcess.execAsync(GROUP_BUILDER_DIR + "/" + ops.commandValue)
```

Comment:
At this point, `ddg-screen-diff` does have the metadata JSON, but I couldn't think of a flexible way to expose that using a CLI. So I thought the next best thing would be to make it easier for people in the company to write their own scripts.

It's not ideal to grab that information twice, I know, but I prefer to leave things flexible.